Chainlit/chainlit
# Set commit message convention for chainlit and literal

## Changes

- Add basic commitlint support to enforce conventional commit

## How to test?

- `pnpm install`
- Add a dummy change
- Test a commit `git commit -m "foo"`
- Notice that you get an error
- Test a valid commit `git commit -m "chore: test dummy commit"`
- Notice that it works
